samples/augment-reality-to-show-tabletop-scene/README.md

```diff
@@ -0,0 +1,42 @@
+# Augment reality to show tabletop scene
+
+Use augmented reality (AR) to pin a scene to a table or desk for easy exploration.
+
+![Image of augment reality to show tabletop scene](augment-reality-to-show-tabletop-scene.png)
+
+## Use case
+
+Tabletop scenes allow you to use your device to interact with scenes as if they are 3D-printed model models sitting on your desk. You could use this to virtually explore a proposed development without needing to create a physical model.
+
+## How to use the sample
+
+Once the scene loads, you'll see a feed from the camera. Wait for ArCore to highlight a plane and tap on one to place the scene. With the scene placed, you can move the camera around the scene to explore.
+
+NOTE: [Google Play Services for AR](https://play.google.com/store/apps/details?id=com.google.ar.core) must be installed to run this app.
```

Comment:
The code in `MainActivity` should do this automatically. Is that worth noting here?